selection_sort.go

```diff
@@ -0,0 +1,16 @@
+package sort
+
+func selectionSort(arr []int) []int {
+	for i := 0; i < len(arr)-1; i++ {
+		minIdx := i
+		for j := i; j < len(arr); j++ {
```

Comment:
The inner loop starting from `i` is correct, but consider adding a comment to explain why the inner loop starts from `i` and not from `0`.